src/Raven.Server/RavenServer.cs

```diff
@@ -279,16 +279,21 @@ void ConfigureKestrel(KestrelServerOptions options)
 
                 if (Configuration.Server.CpuCreditsBase != null ||
                     Configuration.Server.CpuCreditsMax != null ||
-                    Configuration.Server.CpuCreditsThreshold != null)
+                    Configuration.Server.CpuCreditsExhaustionFailoverThreshold != null ||
+                    Configuration.Server.CpuCreditsExhaustionBackgroundTasksThreshold != null)
                 {
                     if (Configuration.Server.CpuCreditsBase == null ||
-                        Configuration.Server.CpuCreditsMax == null ||
-                        Configuration.Server.CpuCreditsThreshold == null)
-                        throw new InvalidOperationException("If Server.CpuCreditsBase, Server.CpuCreditsMax and Server.CpuCreditsThreshold must be specified together");
+                        Configuration.Server.CpuCreditsMax == null)
+                        throw new InvalidOperationException("Both Server.CpuCreditsBase and Server.CpuCreditsMax must be specified");
```

Comment:
Minor - we could use `RavenConfiguration.GetKey(x => .... )` instead of hard coded option names